index.js

```diff
@@ -161,7 +132,6 @@ let endNoteTime = ""
 function playSong(song) {
     song.notes.forEach(
         note => { playNote(note.note, note.duration, note.time_in); endNoteTime = note.time_in + note.duration }
```

Comment:
Could be worth passing the entire note object in instead, then having `playNote` take the note object and deconstruct it